backend/package.json

```diff
@@ -6,6 +6,7 @@
     "@types/node": "7.0.18",
     "@types/passport-local": "1.0.30",
     "@types/sequelize": "4.0.55",
+    "@types/supertest": "2.0.0",
```

Comment:
The `types` are only needed for TS compilation so they are dev dependencies, no?